api/src/test/java/org/apache/cloudstack/api/command/admin/account/CreateAccountCmdTest.java

```diff
@@ -73,7 +73,7 @@ public void testExecuteWithNotBlankPassword() {
         } catch (ServerApiException e) {
             Assert.assertTrue("Received exception as the mock accountService createUserAccount returns null user", true);
         }
-        Mockito.verify(accountService, Mockito.times(1)).createUserAccount(null, "Test", null, null, null, null, null, accountType, roleId, domainId, null, null, null, null);
+        Mockito.verify(accountService, Mockito.times(1)).createUserAccount(createAccountCmd);
```

Comment:
in setUp(), should we set the roleId field too?

Reply:
makes sense, will do

Reply:
It actually doesn't matter for this flimsy test case. We are only verifying that the command is passed to the service, not the handling of it in the service. The service is a mock.